python/tests/conftest.py

```diff
@@ -256,12 +258,27 @@ def lmdb_version_store_prune_previous(version_store_factory):
 
 @pytest.fixture
 def lmdb_version_store_big_map(version_store_factory):
-    return version_store_factory(lmdb_config={"map_size": 2**30})
+    return version_store_factory(lmdb_config={"map_size": 2 ** 30})
 
 
-@pytest.fixture
-def lmdb_version_store_column_buckets(version_store_factory):
-    return version_store_factory(dynamic_schema=True, column_group_size=3, segment_row_size=2, bucketize_dynamic=True)
+@pytest.fixture(scope="function", params=[False])
+def lmdb_version_store_column_buckets(version_store_factory, request):
+    return version_store_factory(
+        dynamic_schema=True, column_group_size=3, segment_row_size=2, bucketize_dynamic=True, reuse_name=request.param
+    )
+
+
+@pytest.fixture(scope="function", params=[False])
+def lmdb_version_store_column_buckets_dynamic_string(version_store_factory, request):
```

Comment:
Please don't create a global fixture that's only used in one test. We need to limit the proliferation of these.

Reply:
These newly added lmdb fixtures also serve as the reminders that "Certain combinations of db setting is needed to be considered".
These combinations are not widely test in other test cases, which I did have found bugs with them, during the development of this project.